src/graphql/query/myLendingStats.graphql

```diff
@@ -34,6 +34,27 @@ query myLendingStats {
 		}
 		userStats {
 			amount_of_loans
+			number_of_loans
```

Comment:
How much extra latency does this add to the query? If it's noticeable, we should create a new query for the new page, so that we don't slow down the old/existing page that uses this query.

Reply:
If filtering and paging are follow-up tickets, this is the remaining open question.

Reply:
![Request Timing](https://github.com/user-attachments/assets/b1d9bd15-0a6b-41d4-8d15-3fd13d6ae850)
![Request Timing](https://github.com/user-attachments/assets/d0ddf30d-247d-40c2-8cac-f6d6f4e70eca)

Reply:
Thanks for the details. I think we might as well create a new query, since we do tend to create unique queries for specific pages, and creating a new query doesn't negatively affect our app maintainability.

Reply:
![Request Timing](https://github.com/user-attachments/assets/f95fbbb0-abe2-4326-8087-5a369b211fb9)
Actually, after reducing the query down to what this new page actually needs, we also brought the response time down significantly on the new page to ~200ms instead of 1.25 seconds.

Reply:
Nice! Yeah some of those gql fields have a lot of monolith processing/code behind them.